packages/core/src/external/aws/s3.ts

```diff
@@ -85,4 +97,41 @@ export class S3Utils {
       Expires: durationSeconds ?? DEFAULT_SIGNED_URL_DURATION,
     });
   }
+
+  async getPresignedUrl({
+    bucket,
+    key,
+    durationSeconds,
+  }: {
+    bucket: string;
+    key: string;
+    durationSeconds?: number;
+  }): Promise<string> {
+    const command = new PutObjectCommand({ Bucket: bucket, Key: key });
+    const presignedUrl = await getPresignedUrl(this.s3Client, command, {
+      expiresIn: durationSeconds ?? DEFAULT_SIGNED_URL_DURATION,
+    });
```

Comment:
IIRC we were limiting the size of the file our customers could upload. Without that, they could upload a file with Gigabytes or more and we would try to process it.

Reply:
This s3 function does not have options for size-limitations.. I think we might have to find another way

Reply:
Omg, PutObjectCommand has the limitations...

Reply:
Done